app/src/main/java/org/oppia/app/parser/StringToFractionParser.kt

```diff
@@ -53,4 +98,16 @@ class StringToFractionParser {
   }
 
   private fun isInputNegative(inputText: String): Boolean = inputText.startsWith("-")
+  /** Enum to store the errors of [FractionInputInteractionView]. */
```

Comment:
Nit: empty newline above for spacing between the enum class and the helper function

Reply:
Done.